cluster/gce/trusty/node.yaml

```diff
@@ -139,6 +139,8 @@ script
 		--configure-cbr0=true \
 		--cgroup-root=/ \
 		--system-container=/system \
+                --runtime-container=/docker-daemon \
+                --resource-container=/kubelet \
```

Comment:
See here: http://upstart.ubuntu.com/cookbook/#code-style. Using tabs is the code style of upstart jobs.